features/css-animations.yml

```
@@ -0,0 +1,51 @@
+name: CSS Animations
+description: The `animation` CSS property applies `@keyframes` animation sequences to transform elements.
```

```suggestion
description: The `animation` CSS property animates an element's style over time, using keyframes described in `@keyframes` rules.
```

Wanted to propose an alt description because I wasn't super keen on using the word "transform", since it could be interpreted as CSS transforms. And because I thought it was important to mention the time component of CSS animations, to separate this feature from the [scroll-driven animations feature](https://web-platform-dx.github.io/web-features-explorer/features/scroll-driven-animations/).